consensus/src/round_manager.rs

```diff
@@ -1257,6 +1257,15 @@ impl RoundManager {
                         qc
                     ))?;
                 if self.onchain_config.order_vote_enabled() {
+                    // This check is already done in safety rules. As printing the "failed to broadcast order vote"
+                    // in humio logs could sometimes look scary, we are doing the same check again here.
+                    if let Some(last_sent_vote) = self.round_state.vote_sent() {
```

Comment:
I think this check here will not remove the warning. `vote_sent` could be None, even though the timeout round is higher than the QC round? Probably need to maintain the highest timeout round in `round_state`?

Reply:
The check is a pretty basic one to remove at least some [warning logs](https://cloud.us.humio.com/telemetry-service-ingest/search?columns=%5B%7B%22type%22:%22field%22,%22fieldName%22:%22@timestamp%22,%22format%22:%22datetime%22,%22width%22:164%7D,%7B%22type%22:%22field%22,%22fieldName%22:%22message%22,%22format%22:%22text%22,%22width%22:827%7D,%7B%22type%22:%22fieldList%22,%22groupByPrefix%22:true,%22header%22:%22all%20fields%22%7D%5D&live=false&newestAtBottom=true&query=Failed%20to%20broadcast%20order%20vote%20for%20QC&showOnlyFirstLine=false&start=7d&tz=America/Los_Angeles&widgetType=list-view). @ibalajiarun has requested this basic check. The safety rules are naturally more comprehensive.

Reply:
I am not saying we need more checks, but this check here may miss some cases. When `self.round_state.vote_sent()` returns None, the check will be ignored and safety rules will still return the warning.

Reply:
Yes. For the majority of the logs in humio (all the logs I've seen at least), we were trying to sign order vote for the same round as the highest time round. My understanding is that, this basic check would try to eliminate warn humio logs for only that case.

Reply:
Okay.